26/upgrade.html

```diff
@@ -41,6 +83,48 @@ <h5><a id="upgrade_260_notable" href="#upgrade_260_notable">Notable changes in 2
     </li>
 </ul>
 
+<h4><a id="upgrade_2_5_0" href="#upgrade_2_5_0">Upgrading from 0.8.x, 0.9.x, 0.10.0.x, 0.10.1.x, 0.10.2.x, 0.11.0.x, 1.0.x, 1.1.x, 2.0.x or 2.1.x or 2.2.x, 2.3.x, 2.4.x to 2.5.0</a></h4>
```

Comment:
Whatever we decide for 2.6.0, we could maybe apply here too.

Reply:
Ack